### PR TITLE
Add pypy ci

### PR DESCRIPTION
**Pull request by dannyjacobs**
This repo is now installable from pipy. This is the preferred installation method for normal users and testing. This PR also adds a github action which causes tagged versions to be built and uploaded to pipy.

### CODE REVIEW

File: .github/workflows/publish-to-test-pypi.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          conda list
+          PYVER=`python -c "import sys; print('{:d}.{:d}'.format(sys.version_info.major, sys.version_info.minor))"`
+          if [[ $PYVER != ${{ env.PYTHON }} ]]; then
+            exit 1;
+          fi
+
+
+      - name: Build a binary wheel and a source tarball
+        run: |
+          python -m pep517.build --source --out-dir dist/ .
```
Comment (steven-murray) [inline .github/workflows/publish-to-test-pypi.yaml:42]: apparently you should just use `python -m build .` these days

File: .github/workflows/publish-to-test-pypi.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+      - name: Publish to PyPI
+        if: startsWith(github.event.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
```
Comment (steven-murray) [inline .github/workflows/publish-to-test-pypi.yaml:47]: is it still master, or is it main?

**Review by steven-murray — COMMENTED** [review]
Almost.

File: .github/workflows/publish-to-test-pypi.yaml (hunk 2 above)
Comment (bhazelton) [reply .github/workflows/publish-to-test-pypi.yaml:47]: It's master.

File: .github/workflows/publish-to-test-pypi.yaml (hunk 1 above)
Comment (dannyjacobs) [reply .github/workflows/publish-to-test-pypi.yaml:42]: done

**Review by bhazelton — APPROVED** [review]
seems reasonable to me.

**Pull request merged**